fuel/app/config/routes.php

```diff
@@ -34,7 +34,14 @@
 	'preview/(:alnum)(/.*)?'               => 'widgets/preview_widget/$1',
 	'preview-embed/(:alnum)(/.*)?'         => 'widgets/play_embedded_preview/$1',
 	'embed/(:alnum)(/.*)?'                 => 'widgets/play_embedded/$1',
-	'lti/assignment?'                      => 'widgets/play_embedded/$1', // legacy LTI url
+	'lti/assignment?' => function () {
```

Comment:
We don't use this much in Materia, so this may be difficult to find in the future.  BUT, Fuel lets us add a function here to determine what controller receives a request.  This is used to support lti implementations that only send requests to a single Materia url.  To support this, lti/assignment will allow picker or content selection requests as well as widget play requests.